src/util/listener-collection.js

```diff
@@ -40,7 +40,7 @@ export class ListenerCollection {
    * @param {Target} eventTarget
    * @param {Type} eventType
    * @param {(event: EventType<Target, Type>) => void} listener
-   * @param {AddEventListenerOptions} [options]
+   * @param {AddEventListenerOptions & { useCapture?: boolean }} [options]
```

Comment:
I don't know why TypeScript doesn't include more properties in its  [`AddEventListener` interface](https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts#L25-L29), but it doesn't.

Reply:
According to the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#syntax) the property is called `capture` rather than `useCapture`. `useCapture` was the name of the old API. Browser support for `capture` as a property looks good.

Reply:
I don't know why this is missing from TS types. It was added to the spec quite a long time ago.